upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -131,5 +131,22 @@ func (b *BootstrapChannelBuilder) buildManifest() (*channelsapi.Addons, map[stri
 		manifests[key] = "addons/" + location
 	}
 
+	if b.cluster.Spec.Networking.Calico != nil {
+		key := "networking.calico"
```

Comment:
Guys I think this is the last change!  I reviewed with @justinsb, who is helping me learn the api stuff.